packages/console/src/browser/ansi-console-item.tsx

```diff
@@ -17,7 +17,11 @@
 import * as React from 'react';
 import { ConsoleItem } from './console-session';
 import { Severity } from '@theia/core/lib/common/severity';
-import Anser from 'anser';
+
+// TODO: Remove this workaround once issue is fixed by anser.
+// REF: https://github.com/IonicaBizau/anser/issues/52
+import AnserTyping from 'anser'; // typing only
```

Comment:
have you tried `const Anser: import('anser').default = require('anser');` to reuse existing typings

Reply:
Thanks for the tip. This one liner essentially does what I did with two line. Thanks!